event/manager.go

```diff
@@ -354,7 +357,7 @@ func handlePendingEvents() {
 		event := *pendingEvents.GetFront()
 		err := handleEvent(&event)
 		if err != nil {
-			if err == Err_Unable_Support {
+			if err == resource.Err_Unable_Support {
```

Comment:
The name of this error `Err_Unable_Support` is not appropriate. Maybe we could use ErrUnableSupport or something else?